### PR TITLE
[4.0] pre-update check

### PR DESCRIPTION
**Pull request by brianteeman**
when moving from bs2 to bs5 the label-* classes were changed to alert-* classes

however there is no such class as alert-important - it should be alert-danger

### before
image

### after
image

### CODE REVIEW

File: administrator/components/com_joomlaupdate/tmpl/joomlaupdate/default_preupdatecheck.php [hunk 1]
```diff
@@ -23,19 +23,19 @@
 $compatibilityTypes = array(
 	'COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_RUNNING_PRE_UPDATE_CHECKS' => array(
 		'class' => 'alert-default',
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/tmpl/joomlaupdate/default_preupdatecheck.php:25]:
```suggestion
		'class' => 'alert-secondary',
```

Comment (brianteeman) [reply administrator/components/com_joomlaupdate/tmpl/joomlaupdate/default_preupdatecheck.php:25]: beyond the scope of this PR. Please create a new issue/pr for that

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f216014e8cfafbdb64b45d9cc3b10e89a8fcd30b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33393">issues.joomla.org/tracker/joomla-cms/33393</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on f216014e8cfafbdb64b45d9cc3b10e89a8fcd30b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33393">issues.joomla.org/tracker/joomla-cms/33393</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33393">issues.joomla.org/tracker/joomla-cms/33393</a>.</sub>

**Comment by chmst** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thank you

**Pull request merged**